timescaledb/tutorials/grafana/grafana-timescalecloud.md

```diff
@@ -0,0 +1,60 @@
+# Connect TimescaleDB and Grafana
+Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
+source plugins that allow you to query and visualize data from a compatible
+database. To add a data source in Grafana you must be signed as a user with
+organization administration role privileges.
+
+To connect Grafana with TimescaleCloud database instance, install Grafana. For
+information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+
+This section shows you how to connect TimescaleCloud database instance as data
+source in [Grafana][grafana-homepage].
+
+## Configure TimscaleDB as a data source
+To configure TimscaleCloud database as data source you need the create a
+TimescaleDB service and then configure TimescaleDB as the data source in
+Grafana.
+
+<procedure>
+
+### Creating a TimescaleDB service
```

Comment:
Should we have install instructions in here, or just point people to the install instructions? I'm leaning towards the latter, because repeating them is creating a maintenance burden. Or you could format it as a partial.

Reply:
Yes @charislam !!!! I was contemplating on that and noticed that the current install instructions does not provide a step to save the Download cheat sheet and that step is crucial for this connection to work.